vmdb/app/models/brand.rb

```diff
@@ -0,0 +1,62 @@
+class Brand
+  attr_accessor :customer_name
+  attr_accessor :vmdb_name
+  attr_accessor :custom_logo
+
+  def initialize(attrs = {})
+    attrs.each do |n, v|
+      public_send("#{n}=", v)
+    end
+  end
+
+  def exist?
+    customer_name && vmdb_name && custom_logo
+  end
+
+  def class_text
+    custom_logo ? "whitelabeled" : ""
+  end
```

Comment:
@Fryguy: we could put such stuff into some presenter or use decorator pattern, if there's more logic dealing with the presentation, but for this small stuff, I'd say that putting it into the model is ok.